.storybook/pages/CoursePlannerEdit/ProjectCard/ProjectCard.tsx

```diff
@@ -110,7 +110,7 @@ export const ProjectCard = ({
         <Heading
           as={headingAs}
           className={styles['project-card__title']}
-          size="body-sm"
+          size="title-xs"
```

Comment:
`body-sm` may have changed in design while this was under way, so picked a near neighbor for this demo.

Reply:
Makes sense and `body-sm` prob being removed as part of Heading ticket anyways